### ISSUE TITLE
Mechanical documentation is depressingly poor and/or nonexistent.

### ISSUE THREAD

**Issue by fake-name** [raspberrypi/documentation#137]
The Model A+ has no mechanical documentation whatsoever. Based on some image comparisons, it *looks* like it's basically a truncated model B+, but this must be clearly spelled out, if so.

The location of the pin 1 on all the connectors *must* be documented clearly in the mechanical drawings. This is critical for many situations where you're designing an board that mates with the Pi, and having to scour third-party blogs to check images is not an acceptable option (Seriously, the only way I can be **certain** I have the connector orientation correct is to look at product pictures on *spark-fun*. WTF?).

Basically, the only way to get accurate measurements seems to be to actually buy a Pi, and use calipers, which is absurd. If you're not going to bother putting together decent docs, at least release the PCB files so I can make mechanical drawings myself.

Lastly, any sane PCB package these days should be able to export decent STEP models of the board, assuming you're properly managing your component models, so that should be only a few clicks, and would be one way I could do the mechanical verification I need myself.
